src/main/java/converter/ConstantRateProvider.java

```diff
@@ -0,0 +1,21 @@
+package converter;
+
+import converter.interfaces.IRateProvider;
+
+import java.math.BigDecimal;
+
+/**
+ * There are no requirements for RateProvider implementation
```

Comment:
There are no requirements for RateProvider implementation but interfaces without the implementation are considered a code smell sometimes, so I'd better have one